estimagic/tests/visualization/test_comparison_plot.py

```diff
@@ -0,0 +1,404 @@
+import json
+
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+from estimagic.visualization.comparison_plot import _add_color_column
+from estimagic.visualization.comparison_plot import _add_dodge_and_binned_x
+from estimagic.visualization.comparison_plot import _add_plot_specs_to_df
+from estimagic.visualization.comparison_plot import _create_bounds_and_rect_widths
+from estimagic.visualization.comparison_plot import _determine_figure_height
+from estimagic.visualization.comparison_plot import _determine_plot_heights
+from estimagic.visualization.comparison_plot import _df_with_all_results
+from estimagic.visualization.comparison_plot import _find_next_lower
+from estimagic.visualization.comparison_plot import _find_next_upper
+from estimagic.visualization.comparison_plot import _flatten_dict
+from estimagic.visualization.comparison_plot import _prep_result_df
+
+# ===========================================================================
+# FIXTURES
+# ===========================================================================
+
+
+@pytest.fixture()
```

Comment:
The brackets are not necessary.

Reply:
0846b2d5aa68a1880f930eade981bce6930216aa.